docs/openAPI.json

```diff
@@ -6697,7 +6661,6 @@
               "purchaseInformationURL": "",
               "shortVersionTitle": "",
               "shortVersionTitleInHebrew": "",
```

Comment:
I think the last property cannot have a comma following it. 

i.e. all cases like this need to shift from (current)

```
 "shortVersionTitle": "",
 "shortVersionTitleInHebrew": "",
}
```

to 

```
 "shortVersionTitle": "",
 "shortVersionTitleInHebrew": ""
}
```

Do you see where the comma was deleted? I know this is super nitpicky, my apologies! Unfortunately, it's not passing validation this way...